www/src/pages/blog/astro-repl.astro

```diff
@@ -0,0 +1,33 @@
+---
+import { Markdown } from 'astro/components';
+import BaseHead from '../../components/BaseHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+
+let title = 'Introducing the Astro REPL';
+let description = 'The power of Astro, inside your browser!';
+let publishDate = 'Thursday, Sep 16 2021';
+let permalink = 'https://astro.build/blog/astro-repl/astro-repl.png';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BaseHead title={title} description={description} permalink={permalink} />
+    <link rel="stylesheet" href={Astro.resolve('../../scss/blog.css')} />
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="drew" heroImage="/assets/blog/astro-repl/astro-repl-hero.jpg" publishDate={publishDate}>
+      <Markdown>
+        The Astro team proudly presents the new Astro REPL: write Astro code straight
```

Comment:
Here's my suggested edits, some for style and some to fix some grammar issues. Also, added a link to the Demo Day live stream as a secondary call to action.

Please rewrite/re-edit in your own voice!

        The Astro team proudly presents the new Astro REPL: write Astro code right in your browser. 
        Use this new REPL to try out Astro's HTML component syntax, prototype your next big idea, 
        or debug issues with ease. Under the hood, the REPL is powered by Astro’s brand new 
        WASM compiler (written in Go) that can rebuild your code faster than you can write it.

        Play around with the Astro REPL today at [astro.build/play](https://astro.build/play). 
        To learn more about our new Go-powered compiler, [join us on Discord](https://astro.build/chat) and tune in for [Astro Demo Days](https://www.youtube.com/watch?v=-ExcBJrXOd8) next Monday at 11am PST.

Reply:
I would use "Monday, September 20, 2021" rather than "next Monday", but otherwise like your suggestions!

Reply:
yes good call!